### ISSUE TITLE
Sticky Footer JS Missing

### ISSUE THREAD

**Issue by aibrean** [olefredrik/FoundationPress#1095]
I noticed that the data element for the sticky footer is applied, but the JS to make it sticky isn't in the codebase.

**Comment by colin-marshall** [comment]
I noticed this too but forgot to open an issue or do a PR. It got removed in the update to Foundation 6.4 and it appears the data attribute was forgotten. Thanks for reporting!

**Comment by olefredrik** [comment]
Actually I think it was removed by mistake.

**Comment by JPOak** [comment]
Just for the record if people have issues and do a search. I did have a conflict with stickyfooter.js and some fullscreen sliders when changing screen sizes and responsiveness. It took me a bit to track down as there were no errors. However, things corrected themeselves after removing stickyfooter.js . I had just started with Foundationpress, so wasn't completely familiar with all its workings. Just a headsup.
